community/sustainability_effort.md

```diff
@@ -0,0 +1,61 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) to the project and in return have their PRs prioritized for review.
+
+## Where is help needed?
+
+Help is needed for:
+* reviewing PRs
+* [triaging](../docs/CONTRIBUTING.md#triaging-bugs) new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels
+* responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* responding to questions in [CNCF Slack](https://argoproj.github.io/community/join-slack) on `#argo-workflows` and `#argo-wf-contributors` channels 
+
+## Commitment
+
+Argo Workflows has roles as described in the [Contributing](../docs/CONTRIBUTING.md#roles) document.
+Any code contributor in a designated role or with an open Membership request can sign up to participate in this Incentive System.
+To create a Membership request, create an Issue in the [argoproj](https://github.com/argoproj/argoproj) repository as described in the [Membership](https://github.com/argoproj/argoproj/blob/main/community/membership.md) guide.
+
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member and below: should average a minimum of 8 hours per week of either [authoring PRs](../docs/CONTRIBUTING.md#authoring-prs) or doing any of the activities listed above in ["Where is help needed?"](#where-is-help-needed) section (more time is required in this role in order to make sufficient progress)
+- Reviewer and above: should average a minimum of 2 hours per week doing any of the activities listed above in ["Where is help needed?"](#where-is-help-needed) section (highest priority is to ensure that all [`prioritized` PRs](#authoring-prs) have an Assignee)
```

Comment:
```suggestion
- Reviewer and above: should average a minimum of 2 hours per week performing any activities listed above in the ["Where is help needed?"](#where-is-help-needed) section
```
- same as above on "performing" and simplifying sentence
- I added a prioritization order in a suggestion to the "Where is help needed?" section, so I think we can remove the parenthesized part with that

Reply:
modified the first bullet, but there's an outstanding question on the second so I left it as is